docs/fsharp/tutorials/asynchronous-and-concurrent-programming/async.md

```diff
@@ -1,213 +1,382 @@
 
-  Although it operates very differently, `Async.Parallel` corresponds to `Task.WhenAll` for the scenario of wanting the results of a set of async jobs when they all complete.
+Signature:
 
-### Differences
+```fsharp
+task: Task<'T>  -> Async<'T>
+```
 
-* Nested `let!` is not allowed, unlike nested `await`
+When to use:
 
-  Unlike `await`, which can be nested indefinitely, `let!` cannot and must have its result bound before using it inside of another `let!`, `do!`, or `use!`.
+* When you are consuming a .NET API that returns a <xref:System.Threading.Tasks.Task%601> within an F# asynchronous computation.
 
-* Cancellation support is simpler in F# than in C#/VB.
+What to watch out for:
 
-  Supporting cancellation of a task midway through its execution in C#/VB requires checking the `IsCancellationRequested` property or calling `ThrowIfCancellationRequested()` on a `CancellationToken` object that’s passed into the async method.
+* This call will allocate an additional `Async<'T>` object, which can increase overhead if it is used often.
 
-In contrast, F# async workflows are more naturally cancellable. Cancellation is a simple three-step process.
+### Async.Catch
 
-1. Create a new `CancellationTokenSource`.
-2. Pass it into a starting function.
-3. Call `Cancel` on the token.
+Creates an asynchronous computation that executes a given `Async<'T>`, returning an `Async<Choice<'T, exn>>`. If the given `Async<'T>` completes successfully, then a `Choice1Of2` is returned with the resultant value. If it raises an exception before it completes, then a `Choice2of2` is returned with the raised exception. If it is used on an asynchronous computation that is itself composed of many computations, and one of those computations throws an exception, then encompassing computation will be stopped entirely.
 
-Example:
+Signature:
 
 ```fsharp
-open System.Threading
+computation: Async<'T> -> Async<Choice<'T, exn>>
+```
 
-// Create a workflow which will loop forever.
-let workflow =
+When to use:
+
+* When you are performing asynchronous work that may fail with an exception and you want to handle that exception in the caller.
+
+What to watch out for:
+
+* When using combined or sequenced asynchronous computations, the encompassing computation will fully stop if one of its "internal" computations throws an exception.
+
+### Async.Ignore
+
+Creates an asynchronous computation that runs the given computation and ignores its result.
+
+Signature:
+
+```fsharp
+computation: Async<'T> -> Async<unit>
+```
+
+When to use:
+
+* When you have an asynchronous computation whose result can be discarded.
+* In conjunction with `Async.Start` or other functions that accept an `Async<unit>`.
+
+What to watch out for:
+
+* If you must use this because you wish to use `Async.Start` or another function that requires `Async<unit>`, consider if discarding the result is okay to do. Discarding results just to fit a type signature should not generally be done
```

Comment:
```suggestion
- If you must use this because you wish to use `Async.Start` or another function that requires `Async<unit>`, consider if discarding the result is okay to do. Discarding results just to fit a type signature should not generally be done.
```